.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+### Description 
+_Add comprehensive description of proposed changes_
+
+
+Fixes # - _issue number(s) if exists_
+
+### Type of change
+
+_Choose one or multiple, leave empty if none of the other choices apply_
+_Add respective label(s) to PR if you have permissions_
```

Comment:
Hmm...during parsing these two lines are gathered into one line, please try to add empty line between

Reply:
Added